### ISSUE TITLE
Test and evaluate the quality of response after docs formatting issues are fixed

### ISSUE THREAD

**Issue by sunank200** [astronomer/ask-astro#188]
After docs formatting issues are fixed we need to run the same [test plan](https://www.notion.so/astronomerio/Ask-Astro-questionnaire-4dfa9e4c55514ec48a83dda2be4e55d2) again and evaluate the results.

Depends on: #184

**Comment by sunank200** [comment]
This has been already tested and merged into the main branch at [184](https://github.com/astronomer/ask-astro/issues/184)
